tutorial.ipynb

```diff
@@ -24,7 +23,7 @@
         "colab_type": "text"
       },
       "source": [
-        "<a href=\"https://colab.research.google.com/github/rohskopf/FitSNAP/blob/minor-updates/tutorial.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+        "<a href=\"https://colab.research.google.com/github/rohskopf/FitSNAP/blob/collected-changes/tutorial.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```

Comment:
Why is the tutorial pointing to your personal branch?

Reply:
Because Colab only allows you to upload to a GitHub repo that you own, but I think changing this href link should point it to the main repo. We'll try it out and see what happens.

Reply:
Alright, lets see with this little experiment